src/state.rs

```diff
@@ -1,52 +1,132 @@
-use super::gui::GUI;
+use super::gui::Gui;
+// Move these to preludes later
+use super::memory::MemoryManagers;
```

Comment:
The `super::memory` and `memory` gets a bit confusing here. I gather the first is the local mod, while the latter is the crate. Maybe consider renaming one of the two for clarity (the local mod could be `sosmemory` for example).